analysis/organize_data.R

```diff
@@ -99,21 +98,6 @@ summary(ipumsdata$racecombo)
 ipumsdata$age[ipumsdata$age==0] <- NA
 summary(ipumsdata$age)
 
-ipumsdata$marriage[ipumsdata$marriage==0] <- NA
-summary(ipumsdata$marriage)
-
-ipumsdata$employment[ipumsdata$employment==0] <- NA
-summary(ipumsdata$employment)
-
-ipumsdata$ihs[ipumsdata$ihs==0] <- NA
-summary(ipumsdata$ihs)
-
-ipumsdata$privatehi[ipumsdata$ihs==0] <- NA
-summary(ipumsdata$privatehi)
-
-ipumsdata$anyhins[ipumsdata$ihs==0] <- NA
-summary(ipumsdata$anyhins)
-
```

Comment:
There is no need for this. With your bracket and boolean approach you start with NA values unless they go elsewhere. Generally, it should always be handled by either the bracket and boolean or nested ifelse recoding.